### PR TITLE
📖Add dev oriented doc for the conformance tests

### PR DESCRIPTION
**Pull request by dims**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: docs/conformance.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+```
+
+## Running the conformance tests
+```
+hack/ci/e2e-conformance.sh
+```
+
+## How to cleanup if you used SKIP_CLEANUP earlier
+```
+hack/ci/e2e-conformance.sh
```
Comment (detiber) [inline docs/conformance.md]: Is this supposed to be the same as above, should it at least tell the user to run `unset SKIP_CLEANUP`?

Comment (dims) [reply docs/conformance.md]: added more words and added `unset` as well

**Pull request merged**